Class-7-(Interfaces).md

```diff
@@ -0,0 +1,19 @@
+# Class 7: Interfaces and Polymorphism
+
+## In the prep work this week, the students learned
+1. What polymorphism is.
+1. What interfaces are and the difference between interfaces and abstract classes.
+1. Different interfaces that are a part of Java and how they can be used in our code.
+1. Best practices around unit tests and interfaces.
+
+## Announcements
+1. Check with your course manager for any important announcements.
+
+## Large Group Time (Instructor)
+
+### Class 7 Topics That Require Careful Attention
+1. Polymorphism has a deceptively simple definition, but can be difficult to grasp. Have students share what polymorphism is to them and share examples to drive home this particular pillar of OOP.
```

Comment:
With any highly abstract concept, the more support we offer to the teachers, the better.  I'd add one or two clear, concrete examples here that show polymorphism - maybe one for an object and one for a method?

Reply:
I added some direction on examples to refer to from the book.

Reply:
OK.